x/auth/client/txbuilder/stdsignmsg.go

```diff
@@ -2,22 +2,22 @@ package context
 
 import (
 	sdk "github.com/cosmos/cosmos-sdk/types"
-	"github.com/cosmos/cosmos-sdk/x/auth"
+	"github.com/cosmos/cosmos-sdk/x/auth/types"
```

Comment:
Yo, now that we have a `types` sub-package, this entire file is redundant and obsolete. Let's remove it.

Reply:
k yeah, actually the entire package is redundant... just txbuilder into types